cmd/config.go

```diff
@@ -204,6 +207,39 @@ type TLSConfig struct {
 	CACertFile *string
 }
 
+// Load reads and parses the certificates and key listed in the TLSConfig, and
+// returns a *tls.Config suitable for either client or server use.
+func (t TLSConfig) Load() (*tls.Config, error) {
```

Comment:
Done. This was good, I found a few problems with how I was printing errors in `Load()`.